manifests/params.pp

```diff
@@ -70,41 +61,23 @@
       $clamd_config      = '/etc/clamd.conf'
       $clamd_service     = 'clamd'
       $clamd_options     = {}
-      $clamd_default_options = {
-        'LogFile'                  => '/var/log/clamav/clamd.log',
-        'LogFileMaxSize'           => '0',
-        'LogTime'                  => 'yes',
-        'LogSyslog'                => 'yes',
-        'PidFile'                  => '/var/run/clamav/clamd.pid',
-        'TemporaryDirectory'       => '/var/tmp',
-        'DatabaseDirectory'        => '/var/lib/clamav',
-        'LocalSocket'              => '/var/run/clamav/clamd.sock',
-        'FixStaleSocket'           => 'yes',
-        'MaxConnectionQueueLength' => '30',
-        'MaxThreads'               => '50',
-        'ReadTimeout'              => '300',
-        'User'                     => 'clam',
-        'AllowSupplementaryGroups' => 'yes',
-        'ScanPE'                   => 'yes',
-        'ScanELF'                  => 'yes',
-        'DetectBrokenExecutables'  => 'yes',
-        'ScanOLE2'                 => 'yes',
-        'ScanMail'                 => 'yes',
-        'ScanArchive'              => 'yes',
-        'ArchiveBlockEncrypted'    => 'no',
-      }
 
       #### freshclam vars ####
       $freshclam_config  = '/etc/freshclam.conf'
       $freshclam_options = {}
-      $freshclam_default_options = {
-        'DatabaseDirectory' => '/var/lib/clamav',
-        'UpdateLogFile'     => '/var/log/clamav/freshclam.log',
-        'LogSyslog'         => 'yes',
-        'DatabaseOwner'     => 'clam',
-        'DatabaseMirror'    => ['db.us.clamav.net', 'db.local.clamav.net'],
-      }
     }
+
+    #### Default values OS specific ####
+    $clamd_default_logfile               = '/var/log/clamav/clamd.log'
+    $clamd_default_pidfile               = '/var/run/clamav/clamd.pid'
+    $clamd_default_temporarydirectory    = '/var/tmp'
+    $clamd_default_databasedirectory     = '/var/lib/clamav'
+    $clamd_default_localsocket           = '/var/run/clamav/clamd.sock'
```

Comment:
localsocket should be this on CentOS 7:  LocalSocket /var/run/clamd.scan/clamd.sock
/var/run/clamav does not exist after installing the package.
Did this work for you?